src/main/java/com/sourcery/gymapp/backend/authentication/config/SecurityConfig.java

```diff
@@ -61,10 +62,10 @@ public SecurityFilterChain jwtFilterChain(HttpSecurity http) throws Exception {
                         .requestMatchers(
                                 "/swagger-ui/**",
                                 "/v3/api-docs/**",
-                                "api/auth/register",
+                                "/api/auth/**",
                                 "/actuator/health"
                         ).permitAll()
-                        .anyRequest().authenticated()
+                      .anyRequest().authenticated()
```

Comment:
Noticed these unnecessary formatting changes again. They add noise to the PR and make reviews harder.
Please avoid them in future commits